acceptance/cmd/psql/test.toml

```diff
@@ -0,0 +1,24 @@
+# This acceptance test is disabled on Windows runners because
+# the current argument capturing method does not work on windows-latest GitHub Runner.
+#
+# See PR #3228 for documented attempts to fix this issue:
+# https://github.com/databricks/cli/pull/3228
+GOOS.windows = false
```

Comment:
Did you try executing the bash as is? Why does that fail?

We do execute other bash scripts as part of acceptance tests.